### DISCUSSION TITLE
Why two different contact?

### DISCUSSION THREAD

**Discussion by happyDuke96** [chatwoot/chatwoot#7571] [Q&A]
Hello Chatwoot Team, I'm using the API channel, I want to create my own widget on the client side.
The client creates a contact == enter his contact details,
[https://www.chatwoot.com/developers/api/#tag/Contacts-API/operation/create-a-contact](url)
then creates a conversation == creates a new conversation,
[https://www.chatwoot.com/developers/api/#tag/Conversations-API/operation/create-a-conversation](url)
and send a message == send an existing inbox with an inbox identifier.
[https://www.chatwoot.com/developers/api/#tag/Messages-API/operation/create-a-message](url)
**And in turn, the agent listens from the websocket and opens the message and sends the message with the conversations id received from the websocket. The question is on the client side it’s clear why create a contact, but why agents create a new contact and conversations.**
[https://www.chatwoot.com/developers/api/#tag/Contacts/operation/contactCreate](url)
[https://www.chatwoot.com/developers/api/#tag/Conversations/operation/newConversation](url)

**And please give more information about pricing, read that every month for each agent let’s say I have 10 agents should I pay for each agent? if so I can use the free version of the principle it suits me please explain the difference**

**Comment by jtayrone** [comment]
Could you detail which API you are using? About plans for the Enterprise version, I suggest you check with support directly.

If you prefer, you can use the self-hosted community version, of course if you have the knowledge to do so.

**Reply by happyDuke96** [reply]
I already installed using docker, I use the Channel API, if you read above, I wrote which APIs I use, I need to know why create a new contact on the agent side, because when clients write, a new contact is still created and conversation,agents can easily write there. **ANY HELP IS APPRECIATED**
